app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+  def create
+    @video = Video.find_by(slug: params[:video_id])
+    @review = @video.reviews.new(review_params)
+    @review.user = current_user
+
+    if @review.save
+      flash["success"] = "Review was successfully created"
+      redirect_to video_path(@video)
+    else
+      flash["error"] = "There was a problem saving your review"
```

Comment:
When using `render`, you need to use `flash.now`. `flash` will persist through the `render` as well as the next redirect.